go.sum

```diff
@@ -585,8 +585,6 @@ github.com/gravitational/httprouter v1.3.1-0.20220408074523-c876c5e705a5 h1:qg8F
 github.com/gravitational/httprouter v1.3.1-0.20220408074523-c876c5e705a5/go.mod h1:JR6WtHb+2LUe8TCKY3cZOxFyyO8IZAc4RVcycCCAKdM=
 github.com/gravitational/kingpin v2.1.11-0.20220506065057-8b7839c62700+incompatible h1:wDxNeLUddZKPWdCAvUMw09nz6PJuBx4U6h1cYEUfNSs=
 github.com/gravitational/kingpin v2.1.11-0.20220506065057-8b7839c62700+incompatible/go.mod h1:LWxG30M3FcrjhOn3T4zz7JmBoQJ45MWZmOXgy9Ganoc=
-github.com/gravitational/license v0.0.0-20210218173955-6d8fb49b117a h1:PN5vAN1ZA0zqdpM6wNdx6+bkdlQ5fImd75oaIHSbOhY=
```

Comment:
We should not remove those
It usually happens because when you don't have the `e/` submodule
You have to checkout the submodule before applying any `go mod/get` commands